pyproject.toml

```diff
@@ -15,6 +15,10 @@ ignore = [
     "noxfile.py",
 ]
 
+[tool.isort]
+known_first_party = "env,pybind11_cross_module_tests,pybind11_tests,"
```

Comment:
Normally, these are autodetected, but it's a bit trickier in this project since the compiled modules don't live in the root, so we have to include any top level modules it might have missed (in tests)